README.md

```diff
@@ -360,6 +360,28 @@ And equally easy to create a program that behaves like `cp`:
   ```
 
 
+## Forcing Parsing
```

Comment:
I love that you not only documented the new feature, but also improved the documentation for the existing functionality.